pom.xml

```diff
@@ -171,6 +171,7 @@
         <easymock.version>4.0.2</easymock.version>
         <eclipselink.version>2.7.4</eclipselink.version>
         <ehcache.version>2.6.11</ehcache.version>
+        <exec-maven-plugin.version>1.6</exec-maven-plugin.version>
```

Comment:
I think you need to reference this in the actual plugin itself with `<version>`. Don't want to rely on Maven "magic" pick it up.

Reply:
My IDE marked it as a WARNING; I can add it anyway; let me submit a new patch.

Reply:
Done.